### ISSUE TITLE
How to synchronized Batch Normalization?

### ISSUE THREAD

**Issue by amdcat** [zhanghang1989/PyTorch-Encoding#4]
Thanks for your project.
I find the BN code in encoding/__init__.py, but i don't konw how to use it? Is the code support multi-GPU now?

**Comment by zhanghang1989** [comment]
It is part of the multi-gpu BN, we haven't released the corresponding DataParallel code.

**Comment by amdcat** [comment]
Is there any plan to release the DataParallel code?

**Comment by zhanghang1989** [comment]
My research got delayed due to visa problem. We may release it later this year.

**Comment by zhanghang1989** [comment]
@amdcat I have just released the whole system for sync BN, please check it out. Note that we will release the paper later, current please cite the 'Deep TEN' paper, if using the code. Thanks!

**Comment by mingminzhen** [comment]
I want to ask how to use sync BN. Could you give some example?

**Comment by zhanghang1989** [comment]
Examples will be provided in later version (later this month). We are preparing an arXiv paper.

**Comment by mingminzhen** [comment]
awesome.

**Comment by mingminzhen** [comment]
I am waiting for the example and paper. When are you ready for that?
